.github/workflows/prod.yml

```diff
@@ -0,0 +1,58 @@
+name: Production
+
+on:
+  push:
+    branches:
+      - prod
```

Comment:
The `prod` branch is `master` as it was said in the beginning of the project

Reply:
Fixed in 9334ae67591de17735fa6661f5c912babaaeb934